docs/How-to/Run-a-full-Terra-node/Join-public-network.md

```diff
@@ -104,17 +112,23 @@ terrad status
 
 ### Wait for node to sync
 
-Your node should now be catching up with the network by replaying all the transactions from genesis and recreating the blockchain state locally. This will take a long time, so make sure you've set it up on a stable connection so you can leave while it syncs.
+::: warning Sync times
+Nodes take at least an hour to sync when starting. This wait is normal. Please be patient and wait for your node to sync before troubleshooting.
```

Comment:
They take an hour, to begin sync.
Actual sync can be multiple days depending on hardrive settings. There is some variability here.
Currently it is about 17hrs to sync.

We just want to let people know that it takes awhile to START syncing

Reply:
Ah I see now. Thanks for clarifying.